python/ray/experimental/serve/utils.py

```diff
@@ -0,0 +1,43 @@
+import json
+import logging
+
+from pygments import formatters, highlight, lexers
+
+import ray
+
+
+def _get_logger():
+    logger = logging.getLogger("ray.serve")
+    logger.setLevel(logging.INFO)
```

Comment:
should this be hard-coded?

Reply:
Yes now. but can be configured with later PRs. Added TODOs.